server/src/dtos/model-config.dto.ts

```diff
@@ -25,6 +27,15 @@ export class CLIPConfig extends ModelConfig {
   mode?: CLIPMode;
 }
 
+export class DuplicateDetectionConfig extends TaskConfig {
+  @IsNumber()
+  @Min(0.01)
+  @Max(0.1)
```

Comment:
I'm not mertalev, but from testing I can say 0.1 is too high. So high enough maximum from my testing.

Reply:
Yeah, after a certain point it stops being duplicate detection and starts being a general "cluster my images" page.

Reply:
Here's an example where it's basically just detecting similar photos rather than duplicates/near-duplicates with the setting at 0.1:
![Screenshot_20240509_123846_Chrome](https://github.com/immich-app/immich/assets/30300649/fe3e0e14-ffad-4408-96c9-aba13512d6b1)

Reply:
Ok nice, thanks for the explanation :)

Reply:
The clustering could be really neat as a separate feature down the road!

Reply:
> The clustering could be really neat as a separate feature down the road!

That's the plan, it would work the same way this does. I saw something about it and was thinking of helping make that feature from this one.